### DISCUSSION TITLE
欢迎来到 CarYon 的讨论区

### DISCUSSION THREAD

**Discussion by luosiwei-cmd** [luosiwei-cmd/CarYon#18] [General]
## 👋欢迎光临！

 我们正在使用“讨论”作为与社区其他成员联系的地方。我们希望您： 

* 问您想知道的问题。

* 分享意见。 

* 与其他社区成员互动。 

* 欢迎他人，思想开放。

在这里可以讨论有关 CarYon 的使用和发展建议或者有关一些 OI/ACM 等**算法竞赛**的知识。

**Comment by dddbearzzz** [comment]
你好，问什么我用devC++尝试运行提供的样例也会出错
<img width="2104" height="1130" alt="QQ截图20251223200624" src="https://github.com/user-attachments/assets/7ed274fc-c581-4cdb-a947-b1b5f9df9234" />
